airflow/api/client/__init__.py

```diff
@@ -19,20 +19,8 @@
 
 from __future__ import annotations
 
-from airflow import api
 from airflow.api.client.local_client import Client
 
 
 def get_current_api_client() -> Client:
-    """Return current API Client based on current Airflow configuration."""
-    auth_backends = api.load_auth()
-    session = None
-    for backend in auth_backends:
-        session_factory = getattr(backend, "create_client_session", None)
-        if session_factory:
-            session = session_factory()
-        api_client = Client(
-            auth=getattr(backend, "CLIENT_AUTH", None),
-            session=session,
-        )
-    return api_client
+    return Client()
```

Comment:
`get_current_api_client` is only used in `airflow/cli/commands` but I think this will go away with AIP-81? @bugraoz93

Reply:
Yes, exactly. Those will be gone as part of AIP-81